### DISCUSSION TITLE
How to pass members of list in the allowed voters

### DISCUSSION THREAD

**Discussion by AayushSaini101** [cncf/gitvote#483] [Q&A]
profiles:
  default:
    duration: 1d
    pass_threshold: 51
    periodic_status_check: "1 week"
    allowed_voters:
      teams:
        - tsc_members

I have tsc_members.yml that contains the allowed voters names in this format. Is the correct format if not. What is the format to pass the members explicitly.,

**tsc_members.yml**
```
-aayushmau5
-imabp
-akshatnema
-anandsunderraman
```

**Comment by tegioz** [comment]
Hi @AayushSaini101 👋

The teams section in allowed_voters is expecting a list of GitHub teams (using their team slug).

Hope this helps 🙂
